objects.go

```diff
@@ -729,25 +729,49 @@ func NewHostRecord(
 }
 
 type RecordTXT struct {
-	IBBase `json:"-"`
-	Ref    string `json:"_ref,omitempty"`
-	Name   string `json:"name,omitempty"`
-	Text   string `json:"text,omitempty"`
-	Ttl    uint   `json:"ttl,omitempty"`
-	View   string `json:"view,omitempty"`
-	Zone   string `json:"zone,omitempty"`
-	Ea     EA     `json:"extattrs"`
-	UseTtl bool   `json:"use_ttl"`
+	IBBase  `json:"-"`
+	Ref     string `json:"_ref,omitempty"`
+	Name    string `json:"name,omitempty"`
+	Text    string `json:"text,omitempty"`
+	Ttl     uint32 `json:"ttl,omitempty"`
+	View    string `json:"view,omitempty"`
+	Zone    string `json:"zone,omitempty"`
+	Comment string `json:"comment"`
+	Ea      EA     `json:"extattrs,omitempty"`
+	UseTtl  bool   `json:"use_ttl,omitempty"`
```

Comment:
Please, group this field with 'Ttl' by moving this line upwards. And remove 'omitempty' flag, otherwise there will be no possibility to set it to 'false' - it will be omitted, and thus TTL may not be inherited from the parent zone.

Please remove the 'omitempty' flag whenever possible, to avoid potential problems when emptying a value of an attribute.

Reply:
Updated so arguments for TXT record match the A Record order throughout the code base

omit empty removed from field that shouldn't have it which resolved a TTL issue I had seen in testing